c/example/README.md

```diff
@@ -0,0 +1,42 @@
+## C example for libsbp
+
```

Comment:
Describe more clearly what this is, i.e.
An example C program, intended to be run on a host computer connected to the Piksi, that parses incoming SBP messages from the Piksi and prints some of them to the screen.